librz/config/config.c

```diff
@@ -344,6 +345,18 @@ RZ_API ut64 rz_config_get_i(RzConfig *cfg, const char *name) {
 	return (ut64)0LL;
 }
 
+RZ_API bool rz_config_get_b(RzConfig *cfg, const char *name) {
```

Comment:
I see a problem here as `false` is both an error value but a valid value. We should probably add an argument for the error or make the return value the error and passing the value of the config var through the arg.

Reply:
In general I agree but it would make this API is inconsistent with the `rz_config_get()` and `rz_config_get_i()` APIs.

Reply:
That's why i said we need unit tests and I'm not sure this is the right time to do this... I would first refactor existing API and then add this new one. Is it really necessary for the purpose of pr? Can't you just use get_i for now and do a full refactoring with all the needed tests in a separate pr?

Reply:
It already improves the situation. I can do a separate refactoring changing the RzConfig API everywhere after this one is done. Is this fine for you?

Reply:
Add at least doxygen and basic unit tests for coverage. This is the way to introduce new APIs

Reply:
Ok, will do.